website-custom-views/src/content/help-needed.mdx

```diff
@@ -34,14 +34,6 @@ It is recommended to always keep your browser up-to-date for **security** reason
 
 # Granular permissions for local development
 
-<Info>
-
-This feature is available from version `22.5.0` onwards.
-
-This method requires additional configuration and setup and is only recommended for development teams with prior experience in Custom Views development and installation.
-
-</Info>
```

Comment:
Let's keep this, you can omit the sentence with the version

Reply:
Applied in fb7a811b30d8fb02e7b111509af229d054724206